mod-orders-storage/schemas/receiving_history.json

```diff
@@ -50,16 +50,16 @@
       "type": "string",
       "pattern": "^[a-zA-Z0-9]{5,16}-[0-9]{1,3}$"
     },
-    "poLineOrderFormat": {
-      "description": "The purchase order line format",
-      "type": "object",
-      "$ref": "order_format.json"
-    },
     "poLineReceiptStatus": {
       "description": "The purchase order line receipt status",
       "type": "object",
       "$ref": "receipt_status.json"
     },
+    "pieceFormat": {
+      "description": "The format of the piece",
+      "type": "object",
```

Comment:
with this format as object and the piece.json as String, is the receiving_history_view able to map the values correctly?

Reply:
Noted. Thanks.